api/cli.py

```
@@ -0,0 +1,332 @@
+                "--format",
+                "{{.State.Running}}",
+                "falkordb-cgraph",
+            ],
+            capture_output=True,
+            text=True,
+        )
+
+        if inspect.returncode == 0:
+            if inspect.stdout.strip() == "false":
+                subprocess.run(
+                    ["docker", "start", "falkordb-cgraph"],
```

Ruff (0.15.6)

[error] Starting a process with a partial executable path

(S607)